test/integration/smoke-testing/test_signing_in_and_my_stuff.js

```diff
@@ -0,0 +1,130 @@
+/*
+ * Tests signing in & My Stuff according to smoke-tests at:
+ *
+ * https://github.com/LLK/scratchr2/wiki/Smoke-Testing-Test-Cases
+ *
+ * Note that I am using a user with at least 1 project and at least 1 studio
+ *  - I'm not sure how to handle this more elegantly... b/c I don't want to commit a password?
+ *  - I could supply a password for a Staging user I guess... but then people may want to test
+ *  - on a variety of instances (local, staging, prod)?
+ */
+
+var username = process.env.USERNAME;
+var password = process.env.PASSWORD;
```

Comment:
Since these are going to be global environment variables on things like Travis or any other host we use for smoke testing, could you name them something more specific? E.g., `SMOKE_USERNAME/PASSWORD`? Also would you please document them in the README, in the Configuration section? https://github.com/LLK/scratch-www#configuration